cal-itp/benefits
# Refactor: rename `EligibilityVerifier` to `EnrollmentFlow`

Closes #2282 

- [x] Will wait for and rebase on #2291

## Testing locally

1. Checkout the `main` branch
1. Make sure that your `local_fixtures.json` (or whatever fixture you are using) has `EligiblityVerifiers`
1. Run `bin/reset_db.sh` (now your local DB mirrors the structure of the `dev` environment)
1. Checkout this PR's branch
1. Run `bin/init.sh` (running this PR's migration on top of existing DB structure)

- [ ] For all `EligiblityVerifier`s, confirm that an instance of `EnrollmentFlow` now exists in the database with the same data
- [ ] Flows using claims and API verification should still work
- [ ] `Can X eligibility verifier` permissions are removed
- [ ] `Can X enrollment flow` permissions are added
- [ ] `Cal-ITP` group has `Can [view|edit] enrollment flow` permissions
